pkg/notify/config/types.go

```diff
@@ -790,6 +796,116 @@ func (w *Wechat) Clone() *Wechat {
 	}
 }
 
+type Pushover struct {
+	// The users (Pushover User Keys) to send notifications to.
+	UserKeys       []string
+	PushoverConfig *PushoverConfig
+	Selector       *metav1.LabelSelector
+	userKeyRegex   *regexp.Regexp // for an User Key's validation
+	*common
```

Comment:
The `template` is missing.

Reply:
Resolved: https://github.com/kubesphere/notification-manager/blob/6229e77a79cb5cadc513d62513f96706f9879ebc/pkg/notify/config/types.go#L800